src/components/10-atoms/text/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 7.0.0
+
+Breaking change:
+- Remove variant `semibold`, because this option is not available in the styleguide
+- Remove variant `bold` use new boolean flag `bold`, because bold for all text sizes possible
+- Change attribute `variant` to `size`
+- When tags are placed inside slot(children). You should use the attribute `tagless` so that the wrapping paragraph is not rendered.
```

Comment:
When **HTML** tags are placed inside axa-text's slot as children, the attribute `tagless`should (**must??**) be used so that the default behaviour of wrapping text in a p(aragraph) is suppressed.

Reply:
This rewrite highlights that `tagless` is confusingly named - why less when tags are added??? Find a better name!

Reply:
nowrap!